### DISCUSSION TITLE
The specified bucket does not exist

### DISCUSSION THREAD

**Discussion by nnor1** [Altinity/clickhouse-backup#1026] [Q&A]
Hello everyone and thank you for your support,

we have a problem creating a backup, while data is in cold storage (s3/obs). If there is no data in cold storage, everything works.

Current situation:
- we use OTC and our endpoint looks like this --> https://bucket-name.obs.eu-de.otc.t-systems.com/
- use of Clickhouse with 3 replicas (no shards)
- cold Storage s3/obs (full permissions)
- clickhouse-backup create test --> works well with minio but not with s3/obs

![image](https://github.com/user-attachments/assets/dac9ad70-f0e5-4b99-83e8-68dcf69bca0e)

Do you need more informations? How do I work around/fix this error?

Thanks

**Comment by nnor1** [comment]
Close discussion

**Reply by Slach** [reply]
is your backup finally worked?

which workaround did you apply?

**Comment by nnor1** [comment]
The create backup now works. We had to change the endpoint from https://bucket-name.obs.eu-de.otc.t-systems.com/ to https://obs.eu-de.otc.t-systems.com/bucket-name/
